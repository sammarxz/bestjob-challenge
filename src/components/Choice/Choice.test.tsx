import React from 'react'
import { render } from '@testing-library/react'

import { Choice } from './'

describe('<Choice type="lizard" />', () => {
  it('should render a choice of lizard', () => {
    const { container, getByText } = render(<Choice type="lizard" />)

    expect(getByText('ðŸ¦Ž')).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot(`
      <body>
        <div>
          <div
            class="bg-white text-4xl border border-gray-100 rounded-xl p-8 shadow-lg hover:shadow-xl duration-300 undefined"
          >
            ðŸ¦Ž
          </div>
        </div>
      </body>
    `)
  })
})
