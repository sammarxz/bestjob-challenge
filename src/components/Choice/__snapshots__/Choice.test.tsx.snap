// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Choice type="lizard" /> should render a choice of lizard: 
      <body>
        <div>
          <div
            class="bg-white text-4xl border border-gray-100 rounded-xl p-8 shadow-lg hover:shadow-xl duration-300 undefined"
          >
            ðŸ¦Ž
          </div>
        </div>
      </body>
     1`] = `
<div
  class="bg-white text-4xl border border-gray-100 rounded-xl p-8 shadow-lg hover:shadow-xl duration-300 undefined"
>
  ðŸ¦Ž
</div>
`;
